app/controllers/lending_policy_controller.rb

```diff
@@ -6,11 +6,16 @@ class LendingPolicyController < ApplicationController
   def accept
     if patron.eligible_for_wage_garnishment?
```

Comment:
```suggestion
      # even if the patron is not eligible for wage garnishment (already opted in or some other reason),
      # just redirect them to the thank you path to avoid further user confusion
      redirect_to lending_policy_thank_you_path unless patron.eligible_for_wage_garnishment?
      
```
 Rather than having nested if-else, here changing this one kinda like a guard clause reads a bit better imo, if we change it this way we won't need the else part and delete line 15-18.

Reply:
thanks, it's fixed now